src/editor/Editor.js

```diff
@@ -961,6 +961,7 @@ define(function (require, exports, module) {
         this._codeMirror.on("focus", function () {
             self._focused = true;
             self.trigger("focus", self);
+            self.document._toggleMasterEditor(self);//Added to toggle fulleditors as master editor
```

Comment:
Nit: missing space after `//` in comments. Here and multiple other places. Just look around to see what the comments should like.